packages/wonder-blocks-modal/components/modal-panel.js

```diff
@@ -114,17 +127,22 @@ export default class ModalPanel extends React.Component<Props> {
             light,
             onClose,
             style,
+            testId,
         } = this.props;
 
         const mainContent = this.renderMainContent();
 
         return (
-            <View style={[styles.wrapper, !light && styles.dark, style]}>
+            <View
+                style={[styles.wrapper, !light && styles.dark, style]}
+                testId={testId && `${testId}-ModalPanel`}
```

Comment:
nit: the suffix here is probably overkill.

Reply:
Added that suffix here to avoid duplication with the `testId` set in `ModalDialog`. Maybe `-panel` would be a better name?